src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final int MAX_BALL_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        int number = random.nextInt(MAX_BALL_NUMBER);
+        String color = new ColorSupplier().getRandomColor();

```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. Here, you can create a single instance of `ColorSupplier` outside the `getRandomBall` method and use it to get random colors.

Reply:
Don't create several instances of the same class if you can use only one instance for your purposes. Here, you can create a single instance of ColorSupplier outside the getRandomBall method and use it to get random colors.